agent/src/util/device_plugin_service.rs

```diff
@@ -497,7 +497,6 @@ async fn try_create_instance(
         nodes: vec![dps.node_name.clone()],
         device_usage,
         metadata: dps.device.properties.clone(),
-        rbac: "rbac".to_string(),
```

Comment:
Separate question, should we have some RBAC support for brokers that we deploy? And how about DH?

Reply:
I could see Akri supporting NetworkPolicies for limiting access to our broker Services, as mentioned in [the security proposal](https://github.com/deislabs/akri/blob/main/docs/proposals/security.md#device-access-via-broker-services). What rbac settings would you impose on the broker itself or the DHs?

Reply:
I was wondering in case the broker or DH had some need of their own to access ApiServer, then it would be good to be able to pass some RBAC rules.

Reply:
I think it makes sense to remove this rbac setting until we flush out a scenario for what you are mentioning. These RBAC settings could be applied to the cluster. I dont see why the broker or DH Pods would need to be aware of them. I could see helm templates being updated to support this though.